atlite/resource.py

```diff
@@ -180,3 +180,13 @@ def smooth(velocities, power):
                     "Turbine generates energy at 0 m/s wind speeds".format(p=params))
 
     return turbine
+
+
+turbines = arrowdict({t[:-5]: t[:-5] for t in
+                     os.listdir(os.path.join(os.path.dirname(__file__),
+                               'resources', 'windturbine'))})
```

Comment:
The paths used for initialising the dictionaries should take the current configuration into account. So, use the construct_filepath‘ function and initialize it from some function which is called when the config changes. @euronion pbb can add a hint!

Reply:
For the path to the currently configured windturbine directory, you want to use
```
from . import config
path = construct_filepath(config.windturbine_dir)
```

For updating the function on a config change, you would have to observe the value change (Observer pattern).
I did not design the `config` module with this in mind, I see two ways to make this cleanly

1. Forget about updating the turbines on a config value change.
2. Implement an observer.

Or hack something away, like wrapping this functionality into a function which gets called at strange places.
(Please: don't).

I'd stick with 1. for now and say we make an observer pattern an enhancement request (new issue, low priority).

Reply:
also I suggest using `glob.glob(path+"*.yaml")` instead of `os.path(..)` and then `str.replace(".yaml","")` to avoid conflicts with foreign files and non-existing indices.

Reply:
And if you are using `glob`, then have a look at the Python 3 [pathlib](https://docs.python.org/3/library/pathlib.html) first!